packages/code-snippet/lib/src/code-snippet/components/code-snippet/code-snippet.component.ts

```diff
@@ -36,7 +36,7 @@ export class CodeSnippetComponent implements OnChanges, AfterViewChecked {
 		if (this.processMarkdown) { this.codeSnippet = marked(this.codeSnippet); }
 	}
 
-	public ngAfterViewChecked() {
+	public ngAfterViewInit() {
```

Comment:
Does this re-highlight when the codeSnippet attribute is changed?

Reply:
As mentioned in person, the previous and new behaviour don't support dynamic updating of the highlighting. I will make a separate issue for this functionality.